governance-charter.md

```diff
@@ -78,15 +78,16 @@ Exception Process outline below.
 - Representatives from major organizations that have adopted OpenTelemetry as
   end-users (one representative per organization)
 
-The bootstrap committee explicitly believes that this heuristic will be
+The Governance Committee explicitly believes that this heuristic will be
 inaccurate and not represent the entire community. An exception form will be
 provided for people who have contributed to the project but may not meet these
-criteria. Exception eligibility applications will be approved by the bootstrap
-committee by a simple majority vote. The exception process will be used as data
+criteria. Exception eligibility applications will be approved by the Governance
+Committee by a simple majority vote. The exception process will be used as data
 points for refining the criteria for the future.
 
-Within its first year, the Governance Committee will redefine these criteria
-around a more robust idea of community membership.
+It is the responsibility of the Governance Committee to refine these criteria 
+prior to each election, including adjusting the number of required contributions,
+and adding any additional community memberships that include eligibility.
```

Comment:
I'm not clear what this means

> adding any additional community memberships that include eligibility

maybe can simplify

```suggestion
It is the responsibility of the Governance Committee to refine these criteria 
prior to each election.
```

Reply:
yes, that works even better, thanks!